src/config.lisp

```diff
@@ -0,0 +1,122 @@
+(in-package :emotiq/config)
+
+(defparameter *dns-ip-zt.emotiq.ch* 
+  '((:hostname "zt-emq-01.zt.emotiq.ch"
+     :ip "10.178.2.166"
+     ;;; The ports take the default values if not otherwise specified
+     :gossip-server-port 65002
+     :rest-server-port 3140
+     :websocket-server-port 3145)
+    (:hostname "zt-emq-02.zt.emotiq.ch"
+     :ip "10.178.0.71")
+    (:hostname "zt-emq-03.zt.emotiq.ch"
+     :ip "10.178.15.71")))
+
+(defparameter *emotiq-conf*
+  (make-pathname :defaults (emotiq/fs:etc/)
+                 :name "emotiq-conf"
+                 :type "json"))
+
+(um:defconstant+ +default-configuration+
+    `((:rest-server
+       . :true)
+      (:rest-server-port
+       . 3140)
+      (:websocket-server
+       . :true)
+      (:websocket-server-port
+       . 3145)
+      (:gossip-server
+       . :true)
+      (:gossip-server-port
+       . 65002)
+      (:genesis-block
+       . "emotiq-genesis-block.json")))
+
+(defun generated-directory (configuration)
+  (let ((host (alexandria:assoc-value configuration :host))
+        (ip (alexandria:assoc-value configuration :ip))
```

Comment:
(1) why do you need to use alexandria:assoc-value? just to have setf'able assoc?  I do not even see where you're setf'ing the assoc. And even if you were, you can just setf the cdr of the result of assoc.
(2) instead of splattering a totally unabstracted advertisement of what your data structure is across the entire module, and maybe beyond, please abstract the access to one place: get-configuration-value or something like that

Reply:
I find 
```
(assoc-value alist key)
```
to be more readable than the nesting
``` 
(cdr (assoc key alist))
```

Reply:
Neither one is winning a beauty contest. Abstract it: (get-configuration-value ...)